service_container.rst

```diff
@@ -133,9 +147,15 @@ the service container *how* to instantiate it:
 
         # app/config/services.yml
         services:
-            app.message_generator:
-                class:     AppBundle\Service\MessageGenerator
-                arguments: []
+            # default configuration for services in *this* file
+            _defaults:
+                autowire: true
+                autoconfigure: true
+                public: false
+
+            # loads services from whatever directories you want (you can update this!)
+            AppBundle\:
+                resource: '../../src/AppBundle/{Service,Command,Form,EventSubscriber,Twig,Security}'
```

Comment:
sorry for being late, but what about giving an absolute path here with `%kernel.root_dir%` for example? I always find those relative paths a bit confusing as it's not always obvious relative to what. Alternatively, maybe we should add an example with _both_